main.go

```diff
@@ -319,6 +317,29 @@ func expandPaths(paths, skip []string) []string {
 	return out
 }
 
+func newPathFilter(skip []string) func(string) bool {
+	filter := map[string]bool{}
+	for _, name := range skip {
+		filter[name] = true
+	}
+
+	return func(path string) bool {
+		base := filepath.Base(path)
+		if filter[base] || filter[path] {
+			return true
+		}
+		return base != "." && base != ".." && strings.ContainsAny(base[0:1], "_.")
```

Comment:
I don't believe so. The old code had this same logic.

I've added a test case so you can confirm.